compiler/ast/ast_parsed_types.nim

```diff
@@ -184,7 +184,7 @@ type
     pnkIteratorDef
     pnkMacroDef
     pnkTemplateDef
-    pnkTypeSection
+    pnkConstDef
```

Comment:
ident def and const def are _very_ similar, why move const def here and why doesn't ident def belong here?

Reply:
I just switch these two, it's `pnkIdentDefs` part of section, structure similar to def